cnxepub/adapters.py

```diff
@@ -418,3 +412,13 @@ def fix_generated_ids(page, content):
 
     for i, node in enumerate(parent):
         parent.set_title_for_node(node, title_overrides[i])
+
+    # For each page in the book, look for links to #auto_ and replace it with
+    # either the value without #auto_{id} or /contents/{id}#target
+    # only fixup links at top, for whole book
+    if depth == 0:
```

Comment:
Are we only fixing the links if the page is directly under a book?  (So not for pages under chapters and units?)